#! /usr/bin/env python

PACKAGE='turtlebot3_auto_camera'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name       Type      Level Description                Default Min   Max
# gen.add("message", str_t,    0,    "The message.",            "hello")
gen.add("clip_hist_percent", double_t, 0, "Clip Hist Percent", 1., 0., 100.)
# gen.add("b",       int_t,    0,    "Second number.",          2,      -100, 100)
# gen.add("enable",  bool_t,   0,    "Enable node operation.",  True)

exit(gen.generate(PACKAGE, "turtlebot3_auto_camera", "ImageCompensationParams"))